### ISSUE TITLE
[Feature Request]: package release binaries

### ISSUE THREAD

**Issue by tuxcanfly** [rollkit/celestia-da#24]
### Implementation ideas

We should create a workflow to build and package binaries with every release.

**Comment by nashqueue** [comment]
cc @MSevey

**Comment by MSevey** [comment]
@tuxcanfly can you elaborate how this celestia-da is used? I thought it was just a library extraction that is imported by rollkit, so it isn't clear to me how a binary would be used.

**Comment by MSevey** [comment]
makes sense, we can copy paste what celestia node and app are already doing for these workflows.
